### ISSUE TITLE
Landmark needed for site-wide navigation

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#384]
Area of site: Homepage
URL: https://hcommons-staging.org

**Description**
The navigational menu does not appear within a landmark region. All site-encompassing content should be contained within a landmark region for ease of accessibility. 

**Recommendation**

Refactor the containing div element (`<nav id="left-panel" class="menu-panel">`) as a `<nav>` element
